test/integration/discovery.go

```diff
@@ -69,6 +69,9 @@ var _ = Describe("Discover Cluster", func() {
 	var discoveryClient *discovery.K8sDiscoveryClient
 
 	BeforeEach(func() {
+		if framework.TestContext.IsOpenShiftTest {
+			Skip("Ignoring all of discovery cases.")
```

Comment:
Are you sure this skip applies to the describe sections of this test spec?

Reply:
yes, adding `skip` in the `BeforeEach` will ignore all of the test spec in the describe sections